lib/flow.js

```diff
@@ -2,8 +2,19 @@
 
 module.exports = (api) => {
 
+  api.metasync.flow = (
```

Comment:
@aqrln sorry they are not identical, `flow(functions)(data, done)` and `composition(fns, done, [data])`